install.sh

```diff
@@ -0,0 +1,48 @@
+# Constants
+VERSION="v0.4.2"
+SIMPLEX_CHAT="simplex-chat"
+
+######################
+PLATAFORM=""
+
+if [ "$(uname)" == "Darwin" ]; then
+	PLATAFORM="macos-x86-64"
+elif [ "$(uname)" == "Linux" ]; then
+	PLATAFORM="ubuntu-20_04-x86-64"
+else
+	echo "Your platform is not suported, try with macos/linux."
+	exit 1
+fi
+
+PLATAFORM_BIN="$SIMPLEX_CHAT-$PLATAFORM"
+######################
+
+######################
+# Detect the shell and the file where to export the path
+FILE_TO_EXPORT=""
+
+if [ -n "$($SHELL -c 'echo $ZSH_VERSION')" ]; then
+	FILE_TO_EXPORT=".zshrc"
+elif [ -n "$($SHELL -c 'echo $BASH_VERSION')" ]; then
+	FILE_TO_EXPORT=".bashrc"
+else
+	echo "Your shell is not suported, try with bash/zsh."
+	exit 1
+fi
+######################
+
+# Check if the directory exists
+SIMPLEX_CHAT_DIR="$HOME/.config/$SIMPLEX_CHAT"
+[ ! -d $SIMPLEX_CHAT_DIR ] && mkdir -p $SIMPLEX_CHAT_DIR
```

Comment:
Hi! Thanks for the PR, a couple of comments from my side.

I think we should switch this to some `bin` folder - you're using it later as a target for downloading the binary, not storing the app's configuration. We currently suggest `~/.local/bin` in the README, and after a brief research into unix directory structure it looks okay, but if you would suggest a better alternative it would be fine as well.

```suggestion
TARGET_DIR="$HOME/.local/bin"
[ ! -d $TARGET_DIR ] && mkdir -p $TARGET_DIR
```

Reply:
I think we can use `~/bin` instead of `~/.local/bin`, which in my case, Archlinux User, didn't even exist until now.

If we use `~/bin` we don't have to export anything to the path because its automatically done, At least in bash, dash and zsh.

https://unix.stackexchange.com/questions/26047/how-to-correctly-add-a-path-to-path/26059#26059
> Some distributions automatically put ~/bin in your PATH if it exists, by the way.